controller.js

```diff
@@ -0,0 +1,111 @@
+
+const Controller = () => {
+    const moudl = moudle()
```

Comment:
there is no reason to shorten the name of your instances. The most important part of coding is clean code and to make sure people can easily read your code. By removing some letters, it only makes it difficult for someone to read. Shorter is not always better. 
`const moudle= Moudle()`

Reply:
no here I have a problem that returns me that I already define Moudle and it will not work same problem for the View